PocketLearner/Common/Assets.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Assets.swift
+//  PocketLearner
+//
+//  Created by 황지우2 on 2023/08/17.
+//
+
+import SwiftUI
+
+// Image Asset
+extension Image {
+    enum imageAssetName: String {
```

Comment:
enum 뒤 타입명은 대문자로 해주세용..!
```suggestion
    enum ImageAssetName: String {
```